GUI/Main.cs

```diff
@@ -187,6 +187,21 @@ public Main(string[] cmdlineArgs, GUIUser User, bool showConsole)
                     Repo.default_ckan_repo.ToString()
                 );
 
+            // Check if there is any other instances already running.
+            // This is not entirely necessary, but we can show a nicer error message this way.
+            try
+            {
+                #pragma warning disable 219
```

Comment:
It is for assigning a variable but never using it, [CS0219](https://msdn.microsoft.com/en-us/library/4kycwe2x(v=vs.90).aspx). Since I know it wouldn't be used, I thought I would reduce the warnings in the output.